hs_core/management/commands/set_aggregations_dirty.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
We need this management command to set the aggregation metadata for all model program aggregations to dirty so that the aggregation meta xml file can be re-generated (to remove the 'model program name' from the meta file) upon bag/aggregation download. Though initially as part of the release deployment we will run this command to set all model program aggregations to dirty, this command can be used in the future to set all aggregations of a specific resource to dirty or all aggregations of a specific type.